CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Contributing to `dbt-trino`
+
+## Getting the code
+
+### Installing git
+
+You will need `git` in order to download and modify the `dbt-core` source code. On macOS, the best way to download git is to just install [Xcode](https://developer.apple.com/support/xcode/).
```

Comment:
What do you think about removing this section? I don't think it's very likely that someone who wants to contribute doesn't have git. The comment about installing git using `Xcode` seems not true, and other OS are not listed.